README.md

```diff
@@ -40,6 +40,37 @@ downloader = HTTPDatasetDownloader()
 downloader.download(source_uri=source_uri, output=data_root)
 
 ```
+### Dataset Explore
+You can explore the downloaded dataset [schema](https://datasetinsights.readthedocs.io/en/latest/Synthetic_Dataset_Schema.html#synthetic-dataset-schema) by using following API:
+
+[Unity Perception](https://datasetinsights.readthedocs.io/en/latest/datasetinsights.datasets.unity_perception.html#datasetinsights-datasets-unity-perception)
+
+
+```python3
+
+from datasetinsights.datasets.unity_perception import (AnnotationDefinitions,
```

Comment:
Let's separate metrics, captures and definitions here into separate code blocks. We can leave out egos and sensors for now as these are less used feature in the current perception package. Also, we should provide explanation in each code block to let user know what to expect.